examples/adafruit_dht/README.md

```diff
@@ -0,0 +1 @@
+@todo
```

Comment:
I am afraid I no longer have any code that uses that in order to generate the examples. Any chance we can try to find some inspiration online?

Alternatively we could include a link to the documentation of these libraries instead of providing an example here.

Reply:
I think we should add an example for each library. As a workaround, we can add a link to the library. I'll try something later.

Reply:
Sounds good.